SomosMasApp/SomosMasApp/App/Shared/Modules/TabBar/ViewController/TabBarViewController.swift

```diff
@@ -11,6 +11,7 @@ class TabBarViewController: UITabBarController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.navigationItem.setHidesBackButton(true, animated: true)
```

Comment:
why? and why animated?

Reply:
This is the back button that we did mentioned today. This return a crash when the user click in it

Reply:
is not a crash is an exit that was putted intentionally because in one of the histories said the it should close the app, anyway is not a big deal

Reply:
If you want to avoid to close the app take out the exit, that's is the solution to your problem